submissions/benjtinsley/test/index.js

```diff
@@ -0,0 +1,28 @@
+var Code     = require('code');
+var Lab      = require('lab');
+var Server   = require('../lib/index.js');
+var Package  = require("../package.json");
+
+var lab      = exports.lab = Lab.script();
+var describe = lab.describe;
+var it       = lab.it;
+var expect   = Code.expect;
+
+describe('Retrieving the version number', () => {
+
+  it('returns the version of the project at /version', (done) => {
+    var options = {
+      method: "GET",
+      url:    "/version"
+    };
+
+    Server.inject(options, (response) => {
```

Comment:
If you wanted to cut this down a bit, you could just pass `'/version'` as the options. Either way is fine.